MdeModulePkg/Library/BaseIpmiCommandLibNull/IpmiCommandLibNetFnSensor.c

```diff
@@ -0,0 +1,48 @@
+/** @file
+  IPMI Command - NetFnApp NULL instance library.
+
+  Copyright (c) 2018 - 2021, Intel Corporation. All rights reserved.<BR>
+  Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.<BR>

```

Comment:
Are these copyrights accurate for the code being added in this change?

Reply:
Changed to Microsoft instead of Intel and AMD